cli/src/auth/oidc.rs

```diff
@@ -244,3 +245,15 @@ pub async fn handle_refresh_tokens(
     let oidc_settings = fetch_oidc_server_settings(auth_info).await?;
     refresh_tokens(&oidc_settings, refresh_token).await
 }
+
+/// Represents the userdata stored for an authentication token.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct UserInfo {
```

Comment:
`serde` will fail the deserialization if any of these fields are not present. But we're only actually using 1 field at the moment. Should we be more permissive here?

Reply:
That seems reasonable, I guess it doesn't hurt to just wrap it in `Option` to show these still might be there but we don`t **need** them? That way when someone else makes changes in the future they know which fields should be present and we don't have to remove them completely.